cmd/project_create.go

```diff
@@ -123,6 +131,7 @@ func init() {
 	projectCreateCmd.Flags().StringP("name", "n", "", "name of the new project")
 	projectCreateCmd.Flags().StringP("group", "g", "", "group name (also known as namespace)")
 	projectCreateCmd.Flags().StringP("description", "d", "", "description of the new project")
+	projectCreateCmd.Flags().StringP("remote", "r", "", "add remote referring to the new project")
```

Comment:
I think we could use one example of that

Reply:
Done! :)